### PR TITLE
WICKET-6884: javadoc:aggregate seems to work locally

### PR DESCRIPTION
**Pull request by solomax**
The only issue I'm observing so far: All links to JDK classes points to Java8 API .....

### CODE REVIEW

File: testing/wicket-threadtest/src/site/site.xml [hunk 1]
```diff
@@ -16,9 +16,4 @@
    limitations under the License.
 -->
 <project name="Wicket Thread Test">
-	<skin>
-		<groupId>org.apache.wicket</groupId>
-		<artifactId>wicket-site-skin</artifactId>
-		<version>1.0-SNAPSHOT</version>
-	</skin>
```
Comment (martin-g) [inline testing/wicket-threadtest/src/site/site.xml:23]: I have no idea what that is!

File: pom.xml [hunk 2]
```diff
@@ -1020,7 +1020,6 @@
 						<failOnError>true</failOnError>
 						<doclint>none</doclint>
 						<additionalJOption>${javadoc.additionalJOption}</additionalJOption>
-						<sourcepath>${project.build.sourceDirectory}</sourcepath>
```
Comment (martin-g) [inline pom.xml:1023]: I should remove this from the jpms branch too!

**Review by martin-g — APPROVED** [review]

File: testing/wicket-threadtest/src/site/site.xml (hunk 1 above)
Comment (solomax) [reply testing/wicket-threadtest/src/site/site.xml:23]: I guess long time ago the site was generated by maven .... :)))

**Comment by martin-g** [comment]
If this fixes the javadoc issue then you could use WICKET-6884 in the title

**Comment by solomax** [comment]
The result of `mvn clean deploy javadoc:aggregate-no-fork` can be checked here: https://ci-builds.apache.org/job/Wicket/job/wicket-9.x/javadoc/

File: pom.xml (hunk 2 above)
Comment (solomax) [reply pom.xml:1023]: Maybe `<source>8</source>` can be safely removed for jpms branch :)

Comment (martin-g) [reply pom.xml:1023]: This is already done: https://github.com/apache/wicket/blob/d0c80aeb774414fcdf11129a98c930ce4f79f25c/pom.xml#L1017

**Pull request merged**